svelte/components/Nft/Nft.svelte

```diff
@@ -47,6 +56,14 @@
   };
 
   $: console.log("Nft", $nft);
+
+  $: chainId && address && account && checkBurnable();
```

Comment:
either load collection onLoad , either add collection Info needed for nft to nftType

Reply:
done in knft-get-metadata